wagtail/admin/views/pages/listing.py

```diff
@@ -11,6 +12,40 @@
 from wagtail.core.models import Page, UserPagePermissionsProxy
 
 
+def apply_filters(filter_query):
+    # filter_query should be of the format 'filter1:value1 filter2:value2'
+    filter_query = filter_query.strip()
+    if not filter_query:
+        return dict()
+    filter_query = filter_query.split(' ')
```

Comment:
I think we should consider sharing a filter function with https://github.com/wagtail/wagtail/blob/571b9e1918861bcbe64a9005a4dc65f3a1fe7a15/wagtail/admin/views/pages/search.py#L67